docs/modules/ROOT/pages/how-tos/configuring/component-nudges.adoc

```diff
@@ -1 +1,87 @@
-= Defining component relationships
\ No newline at end of file
+= Defining component relationships
+
+When an application includes multiple components, their repositories might contain references to image builds of another component. For instance, an application might include multiple child components that, using a Dockerfile `FROM` clause, reference the image build of a common parent.
+
+In such instances, whenever you build a new image of the common parent component, you need to update the references to it. Specifically, you need to copy and paste the pullspec and image digest of that new image to your other source repositories. This process is tedious and error-prone.
+
+However, if you build an application with {ProductName} and make references between components with image digests (`sha256:...`), {ProductName} can automatically generate pull requests to update those references. To use this functionality, simply define the relationships between your components, either in the {ProductName} UI, or in your CLI.
+
+[NOTE]
+====
+By default, {ProductName} only looks for image digest references in Dockerfiles, Containerfiles, and YAML files. However, you can xref:./customizing-the-build.adoc[customize your build pipeline] to adjust that list of files with an annotation. Add the following annotation under `annotations` in your component's `/.tekton/<component-name>-push.yaml` file:
```

Comment:
Nit: it is probably fine just to point them to the push file. We don't support nudging on PRs for now.

Reply:
I think I already point them to the push file, since I say "Add the following annotation...in your component's...*push.yaml* file"
Or am I not following what you meant?

Reply:
I don't know what I was thinking there. But now I am thinking -- can we just explicitly state that this functionality is only enabled for push builds?